packages/ListBox/stories/examples/Showcase.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { action } from "@storybook/addon-actions";
+import { boolean, text } from "@storybook/addon-knobs";
+import { Rule } from "storybook/assets/styles/common.styles";
+import * as characters from "../fixtures/characters";
+import ListBox from "../../src";
+
+const getKnobs = () => ({
+  hasImplicitAll: boolean("hasImplicitAll", false),
+  isDisabled: boolean("isDisabled", false),
+  placeholder: text("placeholder", "Select..."),
+  showFilter: boolean("Show Filter", false),
+  showFooter: boolean("Show Footer", false),
+});
+
+export default function Showcase() {
+  const { showFilter, showFooter, ...listBoxProps } = getKnobs();
+
+  const ListBoxChildren = (
+    <>
+      {showFilter && <ListBox.Filter />}
+      {characters.villians}
+    </>
+  );
```

Comment:
I'm not sure it matters so much for the Showcase source code (more important for Examples, for sure), but it's still a bit more readable to copy this inline.